universe/Pathfinder.cpp

```diff
@@ -1332,13 +1337,15 @@ bool Pathfinder::PathfinderImpl::WithinJumpsOfOthers(
         return false;
     }
 
+    namespace ph = std::placeholders;
+
     // Examine the cache to see if \p system_id is within \p jumps of \p others
     bool within_jumps(false);
     distance_matrix_cache<distance_matrix_storage<short>> cache(m_system_jumps);
     cache.examine_row(system_index,
-                      boost::bind(&Pathfinder::PathfinderImpl::HandleCacheMiss, this, _1, _2),
-                      boost::bind(&Pathfinder::PathfinderImpl::WithinJumpsOfOthersCacheHit, this,
-                                  boost::ref(within_jumps), jumps, others, _1, _2));
+                      std::bind(&Pathfinder::PathfinderImpl::HandleCacheMiss, this, ph::_1, ph::_2),
+                      std::bind(&Pathfinder::PathfinderImpl::WithinJumpsOfOthersCacheHit, this,
+                                boost::ref(within_jumps), jumps, others, ph::_1, ph::_2));
```

Comment:
This needs to be converted to `std::ref`.

Reply:
What specifically, and is that related to the placeholders issue this is addressing?

Reply:
The `within_jumps` parameter of that function should look like:

```
 std::ref(within_jumps)
```

Reply:
> and is that related to the placeholders issue this is addressing?

I'm not sure if you can mix and match `boost::ref` with `std::bind` without problems.